.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
If this change is intentional, could you please put it into a separate commit?

Reply:
Done!